modules/installation-cloudformation-bootstrap.adoc

```diff
@@ -181,7 +181,7 @@ Resources:
         SubnetId: !Ref "PublicSubnet"
       UserData:
         Fn::Base64: !Sub
-        - '{"ignition":{"config":{"replace":{"source":"${S3Loc}","verification":{}}},"timeouts":{},"version":"2.1.0"},"networkd":{},"passwd":{},"storage":{},"systemd":{}}'
+        - '{"ignition":{"config":{"replace":{"source":"${S3Loc}"}},"version":"3.1.0"}}'
         - {
           S3Loc: !Ref BootstrapIgnitionLocation
         }
```

Comment:
Unrelated to your changes, but this curly brace is in red in the preview meaning that there might be a syntax error. Should it be indented with the previous brace?